code-of-conduct.md

```diff
@@ -37,6 +37,8 @@ Examples of unacceptable behavior by participants include:
 * Publishing others' private information, such as a physical or electronic
   address, without explicit permission. This includes any sort of "outing" of
   of any aspect of someone's identity without their consent.
+* Publishing screenshots or quotes, specially from identity channels, without a user's explicit consent.
```

Comment:
I like the emphasis, but as a rule doesn't actually make a distinction: for channels, don't. For ID channels: don't